src/components/Login/login.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+export default class UserLogin extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: '',
+      email: '',
+      reason: ''
+    };
+  }
+
+  handleChange = e => {
+    this.setState({e.target.name: e.target.value})
+  }
+
+  loginUser = () => {
```

Comment:
Will there be added functionality to this later? Or are we leaving it empty because we don't necessarily need a user to fill in any "correct" information?